### PR TITLE
Various Exploration Improvements

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request
Was going to be a bigger set of changes but I'll just do those later.

Improves quality of mining by adding new features such as:
### Mining Vendor
image
Mining vendors are now actual vendors, meaning limited stock and refill requirements. ~~Currently there is no way to obtain these refills ingame, or any way to find these vendors as they are a separate machine to the old ones... but hey, that's a mapper skill issue.~~ Added vendors and refills to various ships.
### Medipens are wack
Survival medipens have been replaced with the survival stimpak, while they *technically* heal more at 80+ healing in brute/burn/oxy within a few seconds. The volatile chems will follow up with a total of toxin/organ/clone 40+ damage over 3 minutes. Making it a great fast source of healing if you don't mind getting cancer later. If you want a taste of that then...
### Grind the legion
Legion cores now yield solus dust when grinded, at least, while active. Once the core becomes inert it will yield nothing so act fast!
### Polar Drip
A variant of the goliath cloak has been added to tribal recipes, it looks styling and has additional cold resistance.  
### Update
Medipens have been buffed/nerfed, turns out it's important to actually test balance changes before making the PR.
Old mining vendors have been replaced with the new vendors in the techweb, vendor refills can be purchased through cargo for 3000cr. 

## Why It's Good For The Game
Content good, boring planets bad.

## Changelog
:cl:
add: Added a cool bear pelt
balance: survival pens now contain meth, I'm not joking
imageadd: new icons for mining vendors
imageadd: new icons for the bear pelt
code: turned mining vendors into normal vendors
code: consequently, normal vendors can now support mining points
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
finally soul dust is real

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -214,6 +214,7 @@
 	else
 		icon_state = "[initial(icon_state)]0"
 
+/* Replaced with variant in whitesands/code/modules/reagents/reagent_containers/hypospray.dm
```
Comment (ZephyrTFA) [inline code/modules/reagents/reagent_containers/hypospray.dm:217]: You can completely remove this if you want. 
We're not keeping modularity as a design core as it has major problems in the future.

**Review by ZephyrTFA — APPROVED** [review]

File: code/modules/reagents/reagent_containers/hypospray.dm (hunk 1 above)
Comment (AsciiSquid) [reply code/modules/reagents/reagent_containers/hypospray.dm:217]: ok, by reviewing the rest of this you can probably tell I gave up on modularity too

**Comment by ZephyrTFA** [comment]
PR author has said this PR is not ready yet via discord. Marking as draft.

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: f789e74004b3acb82690d77398848b240a153b6f
Pull Request: d3b95bd88a31d076a8d74b1dcaeeb10b2f3a2dca
Server: 1dbd0bfb0d4198bd0a0d556e3729419b22e55678

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 89b653c96fc59fa415eeb26666f154fe97f257c1
Pull Request: d3b95bd88a31d076a8d74b1dcaeeb10b2f3a2dca
Server: 9172c1bfefbcd61b7e2dccb8f8cceb235072078e

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 7c3512bd559cf2b1f8e230216ff853c7f118d7ee
Pull Request: d3b95bd88a31d076a8d74b1dcaeeb10b2f3a2dca
Server: 7f432c12878b85f2e0187a8393c33b9d5b197880

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: bf2a3d7daffd8042d3adbe075ec803cd3abd4e03
Pull Request: d3b95bd88a31d076a8d74b1dcaeeb10b2f3a2dca
Server: 68914e28c7ca55ffe87c21c0fc6dbcb9af418093

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Pull request merged**